Haulage/DatabaseSetup.cs

```diff
@@ -0,0 +1,62 @@
+using System.Data.SQLite;
+using System.IO;
+
+public static class DatabaseSetup
+{
+    private static string GetDatabasePath()
+    {
+        // Get the base directory of the application
+        string baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+
+        // Combine the base directory with the relative path to the database
+        string relativePath = Path.Combine(baseDirectory, @"..\..\..\..\..\Database\Haulage.db");
+
+        // Resolve to a full absolute path
+        return Path.GetFullPath(relativePath);
+    }
+
+    private static string connectionString = $"Data Source={GetDatabasePath()};Version=3;";
+
+
+    public static void InitializeDatabase()
+    {
+        if (!File.Exists($@"{GetDatabasePath()}"))
+        {
+            SQLiteConnection.CreateFile($@"{GetDatabasePath()}");
+
+            using (var connection = new SQLiteConnection(connectionString))
```

Comment:
If this was to be put into production using a microsftSQLDatabase would this fail? Could add checks on what type of database is being used and create an appropriate connection

Reply:
Yes this would fail, in theory we should have a Test and Live mode to determine which Database this should be connecting, will propose at next Group Meeting.